server/server.go

```diff
@@ -304,22 +303,15 @@ type txpoolHub struct {
 }
 
 func (t *txpoolHub) GetNonce(root types.Hash, addr types.Address) uint64 {
+	// TODO: Use a function that returns only Account
```

Comment:
Isn't this comment already addressed on line 311? If it is, just remove it.

Reply:
No, this is a reference to how `txpool` receives the data from the state. Instead of having two functions, one for the nonce and another one for the balance it should only receive one with the Account like `jsonrpc` does.